app/models/customer.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+class Customer < User; end
```

Comment:
Why `Customer`? This blog doesn't sell anything %)

Reply:
It doesn't sell anything **YET** :smirk:

Reply:
@sudakov-eugene I think that `Author` is much better name for a regular user in our case.

Reply:
Deal! :1st_place_medal: